dxr/plugins/rust/tests/__init__.py

```diff
@@ -0,0 +1,15 @@
+from dxr.testing import SingleFileTestCase
+
+class RustSingleFileTestCase(SingleFileTestCase):
+    """Test case suited to testing the Rust plugin."""
+    source_filename = 'mod.rs'
+
+    @classmethod
+    def config_input(cls, config_dir_path):
+        config = super(RustSingleFileTestCase, cls).config_input(config_dir_path)
+
+        config['DXR']['enabled_plugins'] = 'pygmentize rust'
+        config['DXR']['disabled_plugins'] = 'clang'
```

Comment:
Shouldn't need this; `enabled_plugins` is comprehensive. `disabled` is useful only when you're doing an "all but these" config with `enabled` equal to `*`.